src/scripts/components/Tooltip.jsx

```diff
@@ -16,12 +19,22 @@ class Tooltip extends React.Component {
   componentDidMount() {
     const tooltipTrigger = this.getTooltipTrigger();
 
-    // Add event listeners
-    // Tooltips are not triggered on touch-devices to not interfere with actionable items
+    /**
+     * Add event listeners for tooltips triggers.
+     * Using Modernizr we detect whether we're dealing with a touch device
+     * in order to attach the correct event listener. The tooltip works with
+     * HOVER on non-touch devices, and CLICK on touch devices.
+     *
+     * @NOTE: We're not using native touch event sequence since touch devices will emulate a
+     * click event and this is the most reliable way to toggle the tooltip.
+     *  */
     if (!Modernizr.touchevents) {
       tooltipTrigger.addEventListener('mouseenter', this.createTooltip.bind(this));
       tooltipTrigger.addEventListener('mouseleave', this.closeTooltip.bind(this));
     }
+    if (Modernizr.touchevents || Modernizr.pointerevents) {
+      tooltipTrigger.addEventListener('click', this.toggleTooltip.bind(this));
```

Comment:
If you define the `toggleTooltip` function using the arrow function, you won't have to bind this function to `this`. 

This line could become:
`tooltipTrigger.addEventListener('click', this.toggleTooltip);`

If you define the function below like:
```
toggleTooltip = (e) => {
  // code here
}
```

The same applies for the other functions where you're doing the same.

Reply:
Good call. I followed the existing code but I like the less verbose approach across the board. 

Will make the changes and merge once complete.